packages/http-client/package.json

```diff
@@ -26,7 +26,6 @@
     "build:js": "swc src -d ./dist --config-file ../../.swcrc --strip-leading-paths",
     "build:types": "tsc --project tsconfig.json --emitDeclarationOnly --skipLibCheck",
     "build": "pnpm build:clean && pnpm build:types && pnpm build:js",
-    "lint": "eslint src --fix",
```

Comment:
All of these are removed as eslint is not configured as a dependency and the workspace is configured to use biome which lints the whole repo so individual package linting is not needed.